opentelemetry-sdk/src/trace/span_processor.rs

```diff
@@ -63,6 +67,10 @@ const OTEL_BSP_MAX_EXPORT_BATCH_SIZE_DEFAULT: usize = 512;
 const OTEL_BSP_EXPORT_TIMEOUT: &str = "OTEL_BSP_EXPORT_TIMEOUT";
 /// Default maximum allowed time to export data.
 const OTEL_BSP_EXPORT_TIMEOUT_DEFAULT: u64 = 30_000;
+/// Default max concurrent exports for BSP
+// TODO jwilm: I omitted the OTEL_ prefix here as this is a non-standard config
+// value. Should it be kept this way or prefix with OTEL_ for consistency?
+const BSP_MAX_CONCURRENT_EXPORTS: usize = 0;
```

Comment:
I think it's reasonable to add `OTEL_` prefix and `_DEFAULT` postfix. 

Also may be set to 1 is more reasonable so that the behavior is consistent with the previous users. For user that encounter span loss they can opt-in the feature

Reply:
Done.